telegram_packs/logos.txt

```diff
@@ -0,0 +1,24 @@
+logos/frogAlpine|💿
+logos/frogAndroid|📱
+logos/frogApple|🍏🍎
+logos/frogArch|💿
+logos/frogCemu|🎮
+logos/frogChrome|🌐
+logos/frogDebian|💿
+logos/frogDiscord|💌📨📨
```

Comment:
- why love?
- why moving letter twice?

Reply:
the moving letter seems to be a copy paste error
I wanted to use the normal ✉️ emoji for the first one
Gonna remove the love letter mapping